client/galaxy/scripts/components/admin/Jobs.vue

```diff
@@ -0,0 +1,371 @@
+<template>
+    <div>
+        <h2 class="mb-3">
+            <span id="jobs-title">Jobs</span>
+        </h2>
+        <Alert :message="message" :variant="status" />
```

Comment:
I think we should avoid using `Alert` if we can use `b-alert` instead. It might be easier to adjust the response in the python-controller.